ot/mpz-ot/src/lib.rs

```diff
@@ -92,6 +92,32 @@ where
     ) -> Result<(), OTError>;
 }
 
+/// A random correlated oblivious transfer sender.
+#[async_trait]
+pub trait RandomCOTSender<T>: ProtocolMessage
+where
+    T: Send + Sync,
+{
+    /// Obliviously transfers the correlations to the receiver.
```

Comment:
I changed "correlation" to "correlated message" for consistency and greater clarity here and in the rest of my suggestions.
```suggestion
    /// Obliviously transfers the correlated messages to the receiver.
```